src/pages/index.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable max-len */
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+import { connect } from 'react-redux';
+import { StripeProvider, Elements } from 'react-stripe-elements';
+import { createSelector } from 'reselect';
+
+import { userSelector } from '../redux/app';
+
+import Spacer from '../components/util/Spacer';
+import DonateForm from '../components/Donation/components/DonateForm';
+import DonateCompletion from '../components/Donation/components/DonateCompletion';
+import PoweredByStripe from '../components/icons/poweredByStripe';
+
+import './index.css';
+
+const propTypes = {
+  email: PropTypes.string,
+  show: PropTypes.bool
+};
+
+const mapStateToProps = createSelector(userSelector, ({ email = '' }) => ({
+  email
+}));
+
+// Stripe public key
+const stripeKey = 'pk_test_gqFA9NGMJiMawvkfLSdpjjxf';
```

Comment:
```suggestion
const stripeKey = 'pk_live_E6Z6xPM8pEsJziHW905zpAvF';
```

Apply this suggestion before merge and after QA.